### ISSUE TITLE
Change behavior of applying a state change (e.g. bold) when no text is selected

### ISSUE THREAD

**Issue by chriskwan** [vistaprint/ArteJS#59]
Currently, if no text is selected and you hit bold, all of the text becomes bold. Instead, if you hit bold, the existing text should be left alone but any text entered from now on should be bold (until you untoggle bold). This is the standard behavior of word processors like Microsoft Word.

**Comment by gshenar** [comment]
this is the standard behavior of other word processors - but it is not the expected behavior of in studio where arte is used. This is probably a topic for discussion

**Comment by chriskwan** [comment]
Yeah in the Studio case, where the text you edit is relatively short, I can see why we would want that behavior. I the proposed behavior would be more ideal when you're editing a large amount of text.

Either way, I think some sort of flag or config would be good to toggle between these.
